### PR TITLE
Revamp Readme.org

### PR DESCRIPTION
**Pull request by xeruf**
Put first things first, apply https://sembr.org and highlight source code blocks.

Btw, the footnotes aren't correctly displayed in Github in case you care ^^

### CODE REVIEW

File: README.org [hunk 1]
```diff
@@ -1,40 +1,55 @@
...
+whose name and definition is set by a customization variable.
+When you export a document,
+the command definition associated with each type of org element
+is exported into the ConTeXt document preamble.
+The element itself is formatted by the associated command.
+This allows documents to be customized
+by either changing the customization variable associated with each element type,
+or by inserting new command definitions into the document preamble
+after the original definitions in order to override them. 
+here are several ways to insert command definitions into the preamble:
```
Comment (Jason-S-Ross) [inline README.org:33]: "here are several ways" should be "Here are several ways" or "There are several ways"

File: README.org [hunk 2]
```diff
@@ -60,31 +74,31 @@ t)
...
   As with ~#+context_header_extra~, snippets can override existing command definitions.
 
-- ~#+context_preset~ :: This keyword allows a new document "Preset" to be
-  defined. A "Preset" is a collection of some literal ConTeXt code to
-  insert into the preamble, a list of snippets, and an "inner template"
-  that specifies overall document structure.
+- ~#+context_preset~ :: This keyword to define a new document /Preset/.
+  A /Preset/ is a collection of some literal ConTeXt code to
+  insert into the preamble, a list of snippets,
+  and an "inner template" that specifies overall document structure.
```
Comment (Jason-S-Ross) [inline README.org:80]: I noticed I wrote this wrong the first time. It doesn't _define_ the preset, it _selects_ it!

How about 
```org
~#+context_preset~ :: This keyword selects the document /Preset/.
  A /Preset/ is a collection of some literal ConTeXt code to
  insert into the preamble, a list of snippets,
  and an "inner template" that specifies overall document structure.
```

**Comment by xeruf** [comment]
updated :)

**Comment by Jason-S-Ross** [comment]
Thanks for you contribution!

**Pull request merged**